### DISCUSSION TITLE
GitHub Actions is currently disabled for your account.

### DISCUSSION THREAD

**Discussion by lodha13** [community/community#78613] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I had an action on one of my repo, which is supposed to be continuously running, I want to test this feature before upgrading my account.

but I noticed that it got cancelled after 360 min and now my account is disabled for action.

Can you please let me know if this is against the policy

**Comment by erickdc7** [comment]
It appears that your GitHub Actions have been disabled after a workflow was cancelled. According to GitHub's policy, Actions can be disabled for a repository or an account if there are signs of abuse or if certain usage limits are exceeded. Here are some steps you can take to resolve this issue:

[1](https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization). **Review GitHub's Usage Policies**:
   Familiarize yourself with the [usage limits](https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization), policies, and quotas for GitHub Actions to ensure that your workflows comply with them. This includes understanding the maximum execution time for jobs and the actions that can lead to disabling.

2. **Check for Automated Abuse or Excessive Resource Consumption**:
   Ensure that your workflows are not engaging in behavior that could be interpreted as abusive, such as excessive API calls or automated processes that consume resources beyond fair use.

3. **Contact GitHub Support**:
   Reach out to GitHub Support for clarification on why the Actions were disabled and to request re-enabling them. Provide details about your workflows and how they comply with GitHub's policies.

4. **Modify Your Workflows**:
   If necessary, modify your workflows to prevent future cancellations or disablements. This may involve optimizing your actions to run within the allowed time frame or reducing resource consumption.

5. **Consider Upgrading Your Account**:
   If you're consistently hitting usage limits, consider upgrading your account to a higher tier that offers more resources for GitHub Actions.

Remember to provide as much detail as possible when contacting support, including any relevant logs or error messages. This will help them assist you more effectively.
